.changeset/poor-boats-walk.md

```diff
@@ -0,0 +1,27 @@
+---
+'@primer/react-brand': minor
+---
+
+CTAForm Component
+
+Example Usage
+
+```jsx live
+<CTAForm>
+    <CTAForm.Input>
+      <FormControl required size="large" fullWidth>
+        <FormControl.Label>Your Work Email Address</FormControl.Label>
+        <TextInput placeholder="name" />
+      </FormControl>
+    </CTAForm.Input>
+    <CTAForm.Confirm>
+      <FormControl required size="large" fullWidth>
+        <FormControl.Label>I agree to the Privacy Policy and Terms of Service</FormControl.Label>
```

Comment:
More a design question, but this is rendering quite heavy compared to how we use this in practice.

E.g.

![Screenshot 2023-02-13 at 14 12 57](https://user-images.githubusercontent.com/13340707/218481360-835605c4-a6ac-433e-a347-044e5417fd5c.png)

cc. @danielguillan

Reply:
It looks good! and that seems to be aligned with the latest Galaxy designs as well. 

<img width="550" alt="Screenshot 2023-02-14 at 12 18 11" src="https://user-images.githubusercontent.com/912236/218721999-1c803567-d558-46e8-8e14-5a24afa4cc84.png">

Reply:
@josepmartins - sorry perhaps wasn't clear. The text in Galaxy is using a non-bold font weight, vs the Primer Brand version, which uses bold. Do we need to ask for Galaxy to be changed? IMO the bold should be limited to shorter phrases that require emphasis, rather than paragraphs like the typical use-case for this component.

Reply:
Update: @josepmartins and I spoke about this separately and agreed to move forward with non-bold font. @josepmartins is planning to update Figma. Reason for this change is that the label text is typically going to be multiline, so emphasising the entire paragraph can make it harder to read.

Reply:
Yep! Just updated the Figma files in case you want to check them out, but in short:

- Font-weight is now using 400 or `--base-text-weight-normal`
- Color is now `--brand-color-text-muted`

Reply:
Is this something that should be updated in the FormControl component or in this example specifically?

Reply:
I'd say only for this example specifically. `FormControl` uses a semibold weight as we assume that labels will be shorter there + it allows a `FormControl.Hint` to get more control over styles and long-length descriptive texts.

<img alt="Screenshot 2023-02-15 at 08 52 26" src="https://user-images.githubusercontent.com/912236/218966824-de86815c-c4f4-4c74-b48a-a5e1c66e1f82.png">

Reply:
Luckily it seems anything we want to create is possible because the label is composable.

Does this mean that my current example is correct in term of its look and use in our documentation?

Reply:
I'd say it's missing the variant="muted" in the Text component

```jsx
<Text size="300" variant="muted">....</Text>
```

Reply:
Perfect I the docs and they should align now.